### PR TITLE
[vernac] Don't allow attributes on print / check

### LINKED ISSUE #11661: Any attribute is accepted on query commands.
#### Description of the problem

These commands are accepted:

```coq
Polymorphic About nat.
#[ universes(polymorphic) ] Check nat.
#[ deprecated(note="") ] Print nat.
#[ canonical ] Compute nat.
```

They probably should raise the `unsupported-attributes` fatal warning instead.

#### Coq Version

Since 8.10.

In 8.9, `#[ polymorphic ] Check nat.` raised `This command does not support Polymorphism`.

### PR DESCRIPTION
**Pull request by ejgallego**
Fixes #11661

### CODE REVIEW

**Comment by Zimmi48** [comment]
Please ignore: this is just a test.

@coqbot: merge now

**Comment by ejgallego** [comment]
@Zimmi48 , IMHO this is simple enough for us to merge and ship in 8.12.1

**Comment by Zimmi48** [comment]
Yep, I agree this is pretty trivial.

File: vernac/vernacentries.ml
Comment (Zimmi48) [inline vernac/vernacentries.ml:2231]: What does it change to have the attribute error inside or outside `VtReadProofOpt`?

**Review by Zimmi48 — APPROVED** [review]
I'll merge tomorrow if @coq/vernac-maintainers are too busy to handle this PR.

File: vernac/vernacentries.ml
Comment (ejgallego) [reply vernac/vernacentries.ml:2231]: It delays the check to actual vernac execution vs the translation phase.

Indeed the translation function is not supposed to produce any side-effects as it could be useful for example for static analysis, so this is a micro bug fix.

**Comment by Zimmi48** [comment]
@coqbot: merge now

**Pull request merged**
